inclusive-design/idrc
# feat: add toggle to allow news and ideas to be archived

Resolves #170.


## Description

As identified in #170, it can be beneficial to mark news or ideas posts as "archived" so that they can remain in source control but be hidden from the public site. This PR adds a switch to the CMS which lets content editors mark news ore ideas posts as "archived", and if they are marked as such they will be excluded from the build.

## Steps to test

1. Run `npm start`.
2. Visit the CMS at `http://localhost:8080/admin`.
3. Toggle the "Archived" switch for a news or ideas post, and the publish the changes.

**Expected behavior:** The news or ideas post should no longer be accessible on the local site.

## Additional information

Not applicable.

## Related issues

Once this is live, https://github.com/inclusive-design/idrc/pull/356 and https://github.com/inclusive-design/idrc/pull/382 can be modified to mark these two items as archived rather than leaving them as drafts, and then the PRs can be merged.
